private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/MavenRepositoryPath.java

```diff
@@ -0,0 +1,38 @@
+package com.github.bazelbuild.rules_jvm_external.resolver;
+
+import com.github.bazelbuild.rules_jvm_external.Coordinates;
+import com.google.common.base.Strings;
+
+public class MavenRepositoryPath {
+
+  private final String path;
+
+  public MavenRepositoryPath(Coordinates coords) {
```

Comment:
Should this just be a method on coordinates?  So you could do `coordinates.getMavenRepositoryPath();` ?

Reply:
Quite possibly. I'll move it.